src/Controls/src/Core/AppThemeBinding.cs

```diff
@@ -8,10 +8,38 @@ namespace Microsoft.Maui.Controls
 {
 	class AppThemeBinding : BindingBase
 	{
+		class AppThemeProxy : Element
+		{
+			public AppThemeProxy(Element parent, AppThemeBinding binding)
```

Comment:
I see the `AppThemeBinding` uses a weak reference for the target: `WeakReference<BindableObject> _weakTarget;` Is this now creating a string reference that might cause issues? Maybe the AppThemeBinding does not need to be weak?

Reply:
the proxy might prevent the AppThemeBinding to GC, but the AppThemeBinding doesn't hold a strong ref to the BO.
in this case, I trust the GC to do the job

Reply:
looks like I was wrong